### PR TITLE
Custom pokemon markers and info

### PR DESCRIPTION
**Pull request by poulzinho**
Some custom markers (pokemon icons) and popups with info

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -29,3 +32,48 @@ var layer = L.tileLayer(tiles, {
...
+var LeafIcon = L.Icon.extend({
+    options: {
+        iconSize:     [50, 46],
+        popupAnchor:  [-7, -20]
+    }
+});
+
+// mark Pokemons
+L.marker([48.262299, 11.669776], {icon: new LeafIcon({iconUrl: 'map/css/images/pokemon/pikachu.gif'})})
+    .addTo(map).bindPopup('Pokemon: <b>Pikachu</b><br/> <br/>' +
```
Comment (sacdallago) [inline app.js]: I suggest you to create a pop-up object, then a marker, and finally bind the pop-up to the marker :)

Comment (poulzinho) [inline app.js]: Done!

**Comment by sacdallago** [comment]
I already saw that you credit all the sources, which is wonderful. Just double check in: are the images/icons free to use?

**Comment by poulzinho** [comment]
It appears that only the written content in pokemondb is protected. We can use the sprites under Fair Use Policy as they are doing now.

> Can I use some content from your site?
> 
> In general, no. All written content, site design and data display/layouts have had a huge amount of research and effort put into them to make as easy to read as possible. Do not steal our content!
> 
> However, **official Pokémon artwork and sprites are not our creations and are used under a Fair Use Policy, so you are free to use them however you like. Each of the Pokémon sprites pages has an easy way to add images to your site or forum post.**

http://pokemondb.net/about

**Comment by sacdallago** [comment]
Thanks @poulzinho

@gyachdav for further reference

**Comment by sacdallago** [comment]
Can this be merged? PRs should not exist for longer than 1-2 days.

The PR should be merged by @gqinami or @ukrainian256 though ;) That's how code-revision works

**Pull request merged**